pom.xml

```diff
@@ -661,6 +663,13 @@
       <plugin>
         <groupId>org.kohsuke</groupId>
         <artifactId>access-modifier-checker</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>enforce</goal>
```

Comment:
Hmm, seems it _was_ being run before, though I am not sure how; https://github.com/kohsuke/access-modifier/pull/3/files#diff-2821f9ce286c3a693f454dae0bfa2ac7R23 seems to be required.